scripts/odt2pdf/odt2pdf.sh

```diff
@@ -18,41 +19,61 @@
 # If you got this error: javaldx failed! Warning: failed to read path from javaldx with no return to prompt when running soffice --headless -env:UserInstallation=file:"/tmp" --convert-to pdf --outdir xxx ./yyy.odt,
 # check that directory defined into env:UserInstallation parameters exists and is writeable.
 
-# shellcheck disable=2046,2086,2268
-
-
-if [ "x$1" == "x" ]
+if [ "$1" = "" ] || [ "$2" = "" ]
```

Comment:
What happen if you run the script without parameters ? Don't you have a syntax error ?
The x is here to avoid that...

Reply:
No, I do not have any script error.

Here's what shellcheck has to say about it:
https://www.shellcheck.net/wiki/SC2268